pkg/net/http/httputils.go

```diff
@@ -110,6 +112,7 @@ func ValidateParameters(params map[string]string) (*QueryHeader, error) {
 		SortOrder:    sortOrder,
 		StartDate:    startDate,
 		EndDate:      endDate,
+		Alias:        alias,
```

Comment:
Validate if alias should be +1
```suggestion
		Aliases:        aliases,
```